haproxy/README.md

```diff
@@ -56,10 +56,12 @@ $ docker run -it --rm --name haproxy-syntax-check haproxy:1.5 haproxy -c -f /usr
 $ docker run -d --name my-running-haproxy my-haproxy
 ```
 
+You may need to publish the ports your haproxy is listening on to the host by specifying the `-p` option, for example `-p 80:80` to publish port 80. Make sure the port you're using is free.
+
 ## Directly via bind mount
 
 ```console
-$ docker run -d --name my-running-haproxy -v /path/to/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:ro haproxy:1.5
+$ docker run -d --name my-running-haproxy -v /path/to/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:ro my-haproxy
```

Comment:
This one doesn't need to be `my-haproxy` since it is using a bind mount rather than doing the `docker build`.